sql/core/src/test/scala/org/apache/spark/sql/scripting/SqlScriptingInterpreterSuite.scala

```diff
@@ -670,7 +1048,7 @@ class SqlScriptingInterpreterSuite extends QueryTest with SharedSparkSession {
       exception = intercept[SqlScriptingException] (
         runSqlScript(commands)
       ),
-      errorClass = "INVALID_BOOLEAN_STATEMENT",
+      condition = "INVALID_BOOLEAN_STATEMENT",
```

Comment:
hmm, strange. Why do you need the changes?

Reply:
It's strange that github caught these changes, as the code is the same on master. I've rebased on master and now the changes are gone